packages/app/components/plugins/SignUp/index.tsx

```diff
@@ -0,0 +1,104 @@
+'use client'
+import { signUp } from '@/lib/actions/events'
+import { useEffect, useState } from 'react'
+import { CredenzaTrigger, Credenza } from '@/components/ui/crezenda'
+import {
+  CredenzaContent,
+  CredenzaDescription,
+  CredenzaTitle,
+  CredenzaHeader,
+  CredenzaFooter,
+  CredenzaBody,
+} from '@/components/ui/crezenda'
+import { useFormStatus, useFormState } from 'react-dom'
```

Comment:
useFormStatus and useFormState are causing build error here. This is a typeScript issue due to the lack of type definition for the new hooks. 
Recommended solutions can be found here, https://github.com/vercel/next.js/issues/56041?ref=nehalist.io#issuecomment-1782957069 We either add ''ts-ignore' above the import or a type declarations

Reply:
yeah its a experimental react feature - can use ts-ignore